extensions/blocks/podcast-player/templates/playlist-track.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * Podcast Title template.
+ *
+ * @package Jetpack
+ */
+
+namespace Automattic\Jetpack\Extensions\Podcast_Player;
+
+// phpcs:disable VariableAnalysis.CodeAnalysis.VariableAnalysis.UndefinedVariable
+
+$track_title    = $attachment['title'];
+$track_duration = ! empty( $attachment['duration'] ) ? $attachment['duration'] : '';
+
+$class = 'jetpack-podcast-player__track ' . $secondary_colors['class'];
+$style = $secondary_colors['style'];
+if ( $is_active ) {
+	$class = 'jetpack-podcast-player__track is-active ' . $primary_colors['class'];
+	$style = $primary_colors['style'];
+}
+
+?>
+
+<li
+	class="<?php echo esc_attr( $class ); ?>"
+	style="<?php echo esc_attr( $style ); ?>"
```

Comment:
Could this be rewritten to not output the `style` tag if there's no `$style`? If you don't have any colors selected, then there's no `$style` and it outputs an empty `style` tag:

<img width="349" alt="Screen Shot 2020-04-01 at 11 27 12 AM" src="https://user-images.githubusercontent.com/967608/78161965-d4179900-740b-11ea-82a4-c31a354d5b0d.png">

Reply:
> Could this be rewritten to not output the `style` tag if there's no `$style`? If you don't have any colors selected, then there's no `$style` and it outputs an empty `style` tag:
> 
> <img alt="Screen Shot 2020-04-01 at 11 27 12 AM" width="349" src="https://user-images.githubusercontent.com/967608/78161965-d4179900-740b-11ea-82a4-c31a354d5b0d.png">

we agree don't mind leaving the empty attributes there. cc @obenland

Reply:
we agree don't mind leave these attributes without values cc @obenland